runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/DefaultJobBundleFactory.java

```diff
@@ -374,17 +374,18 @@ ServerInfo getServerInfo() {
       return serverInfo;
     }
 
-    @Override
-    public void close() throws Exception {
-      try (AutoCloseable envCloser = environment) {
-        // Wrap resources in try-with-resources to ensure all are cleaned up.
```

Comment:
Because this was ignored in the past, I'd keep this comment with an extra warning :) I'll do that during merge.

Reply:
Ah, just saw you moved it. Great 👍 Will still add an extra warning.